docs/product-user-guide/src/main/asciidoc/chap-process-designer.adoc

```diff
@@ -27,15 +27,7 @@ Note that one process definition may contain exactly one process diagram; theref
 . The Object Library (palette) contains groups of BPMN2 elements.
 Details on execution semantics and properties of individual BPMN2 shapes are available in <<_appe_process_elements>>.
 . The Properties panel displays the properties of the selected element. If no element is selected, the panel contains process properties.
-. The editor toolbar enables you, for example, to select an operation to be applied to the Elements on the canvas. It also contains tools for validation, simulation, saving, and others. 
-
-[NOTE]
-====
-To enlarge the Process Designer screen (or any screen while working in Business Central), click on the button shown here: image:6577.png[]
-.
-This will make your current editor fill the entire Business Central screen.
-To go back, simply click the button again.
-====
+. The editor toolbar enables you, for example, to select an operation to be applied to the Elements on the canvas. It also contains tools for validation, simulation, saving, and others.
```

Comment:
The "Elements" should not be capitalized here:
"........to select an operation to be applied to the **Elements** on the canvas."